Lib/test/test_signal.py

```diff
@@ -941,6 +942,101 @@ def handler(signum, frame):
                                 (exitcode, stdout))
 
 
+class StressTest(unittest.TestCase):
```

Comment:
Oh... I dislike such tests, I expect that I will have to fix corner cases in the test itself on BSD, Solaris, etc. I understand why you wrote the test, but I also expect that it will be a pain to "maintain" it :-(